### ISSUE TITLE
Error: Minified exception occurred

### ISSUE THREAD

**Issue by ramoncaldeira** [avocode/avocode#1137]
IMPORTANT: Please try to describe how did it happen.
---

App v. `2.11.0`
Source UID: 56406

```
Error: Minified exception occurred; use the non-minified dev environment for the full error message and additional helpful warnings.
    at invariant (/usr/local/bin/avocode/resources/app.asar/node_modules/react/node_modules/fbjs/lib/invariant.js:35:15)
    at Object.findComponentRoot (/usr/local/bin/avocode/resources/app.asar/node_modules/react/lib/ReactMount.js:764:478)
    at Object.findReactNodeByID (/usr/local/bin/avocode/resources/app.asar/node_modules/react/lib/ReactMount.js:679:23)
    at Object.getNode (/usr/local/bin/avocode/resources/app.asar/node_modules/react/lib/ReactMount.js:162:32)
    at ReactDOMComponent.getPublicInstance (/usr/local/bin/avocode/resources/app.asar/node_modules/react/lib/ReactDOMComponent.js:925:29)
    at Object.removeComponentAsRefFrom (/usr/local/bin/avocode/resources/app.asar/node_modules/react/lib/ReactOwner.js:84:59)
    at detachRef (/usr/local/bin/avocode/resources/app.asar/node_modules/react/lib/ReactRef.js:32:16)
    at Object.ReactRef.detach
```

**Comment by connor-bean** [comment]
Hello @ramoncaldeira -

Could you provide us with more information on what you were doing when this happened? Has it happened again since?

Thanks,
Connor, Avocode Support

**Comment by ramoncaldeira** [comment]
Hi @connor-bean 

Unfortunately I do not remember the details. It might be solved, because it happened along with #1138 and it was fixed on 2.11.1. This error never happened again, after version update.

**Comment by jakubzitny** [comment]
Thanks for the clarification @ramoncaldeira. It makes sense that these two issues were connected to each other. I think we can close this.
